### DISCUSSION TITLE
Encountering error while executing npm cmd

### DISCUSSION THREAD

**Discussion by vedxp** [termux/termux-app#3096] [Q&A]
- I've tried clearing data, but not solved.
- I've no root access.
#### I need help fixing this error:
> ![Screenshot_20221115-192757](https://user-images.githubusercontent.com/87353286/201938071-9d979363-3ba0-41a4-92b2-a490ba819163.png)

**Comment by sk0kanik** [comment]
termux/termux-packages#13293
